driver-sync/src/test/functional/com/mongodb/client/JsonPoweredCrudTestHelper.java

```diff
@@ -372,14 +373,31 @@ BsonDocument getDatabaseListCollectionsResult(final BsonDocument databaseOptions
     BsonDocument getCreateIndexResult(final BsonDocument collectionOptions, final BsonDocument arguments,
                                       @Nullable final ClientSession clientSession) {
         String index;
+        IndexOptions indexOptions = new IndexOptions();
+
+        if (arguments.containsKey("name")) {
+            indexOptions.name(arguments.getString("name").getValue());
+        }
         if (clientSession == null) {
-            index = getCollection(collectionOptions).createIndex(arguments.getDocument("keys", new BsonDocument()));
+            index = getCollection(collectionOptions).createIndex(arguments.getDocument("keys", new BsonDocument()), indexOptions);
         } else {
-            index = getCollection(collectionOptions).createIndex(clientSession, arguments.getDocument("keys", new BsonDocument()));
+            index = getCollection(collectionOptions).createIndex(clientSession, arguments.getDocument("keys", new BsonDocument()),
+                    indexOptions);
         }
         return toResult("result", new BsonString(index));
     }
 
+    BsonDocument getDatabaseCreateCollectionResult(final BsonDocument collectionOptions, final BsonDocument arguments,
```

Comment:
I don't think we should ignore `collectionOptions`.  If the tests define any, we shouldn't ignore them.  If they don't, then let's remove the parameter.

Reply:
Looking at the change in context of the rest of the class, I realize that I was wrong about this request.  The change you made to address it should be reverted.  The only change that seems to make sense is to call the unused parameter `databaseOptions`, to go along with the pattern of other methods that act on a database.

I am not quite sure why we are ignoring these options, but it seems we are already doing it elsewhere.